code/lib/cli/src/detect.ts

```diff
@@ -106,23 +106,29 @@ export function detectFrameworkPreset(
  *
  * @returns CoreBuilder
  */
-export function detectBuilder(packageManager: JsPackageManager) {
+export function detectBuilder(packageManager: JsPackageManager, projectType: ProjectType) {
   const viteConfig = findUp.sync(viteConfigFiles);
-
   if (viteConfig) {
     paddedLog('Detected vite project, setting builder to @storybook/builder-vite');
```

Comment:
While we're at it, let's change this to just "setting builder to vite"

Reply:
done